### PR TITLE
GUVNOR-2933: Guvnor ALA integration

### PR DESCRIPTION
**Pull request by wmedvede**

### CODE REVIEW

**Comment by wmedvede** [comment]
Hello @porcelli, this PR is for the maven build integration we have been chatting about.
Would you mind review.

Thanks.

File: guvnor-bom/pom.xml [hunk 1]
```diff
@@ -164,6 +164,12 @@
       </dependency>
       <dependency>
         <groupId>org.guvnor</groupId>
+        <artifactId>guvnor-ala-build-maven</artifactId>
+        <version>${version.org.kie}</version>
+        <classifier>tests</classifier>
```
Comment (porcelli) [inline guvnor-bom/pom.xml]: @wmedvede i don't think we use classifiers here

Comment (wmedvede) [inline guvnor-bom/pom.xml]: @porcelli I'll double check, but in the past, on other modules, I had to add the "tests" classifier in to be able to use e.g. some TestBase in guvnor, in other test class in kie-wb-commons. And this is basically the case here, I'm interested in using some test classes from guvnor in kie-wb-commons

Comment (psiroky) [inline guvnor-bom/pom.xml]: There are two options for test jars:
 1. `<classifier>tests</classifier>`
 2. `<scope>test</scope>`

We are mixing both in different POMs, so it probably does not matter which one gets used.

Comment (porcelli) [inline guvnor-bom/pom.xml]: @wmedvede k, got what you meant;

Comment (wmedvede) [inline guvnor-bom/pom.xml]: @psiroky I see, thanks. in this case the <classifier>tests</classifier> is being used.

**Pull request closed without merging**